src/oc_erchef/apps/chef_db/src/chef_db.erl

```diff
@@ -258,6 +258,7 @@ update(#chef_cookbook_version{org_id =OrgId} = Record, #context{reqid = ReqId} =
 update(ObjectRec, #context{reqid = ReqId}, ActorId) ->
     case stats_hero:ctime(ReqId, {chef_sql, do_update},
                           fun() -> chef_sql:update(ObjectRec, ActorId) end) of
+        0 -> {error, <<"Record not updated '", ObjectRec/binary, "'.">>};
```

Comment:
It looks like the changes [do_update](https://github.com/chef/chef-server/pull/1383/files#diff-43908ac1f0b634347760bb3d12a9cba8R1100) will return `{error, Reason}` - can we remove this branch from here? 

Same question for `create`, looks like the {ok. 0} return is handled there.

Reply:
there are 2 different code paths that call into either the do_update or update functions, so we need this in both places.

Reply:
yes!